kyo-combinators/shared/src/main/scala/kyo/Combinators.scala

```diff
@@ -417,90 +425,29 @@ extension [A, S, E](effect: A < (Abort[Absent] & S))
       */
     def emptyAbortToFailure(failure: => E)(using Flat[A], Frame): A < (S & Abort[E]) =
         for
-            res <- effect.handleSomeAbort[Absent]()
+            res <- effect.someAbort[Absent].handled
         yield res match
             case Result.Fail(_)    => Abort.get(Result.Fail(failure))
             case Result.Success(a) => Abort.get(Result.success(a))
             case res: Result.Panic => Abort.get(res)
 end extension
 
-class SomeAbortToChoiceOps[A, S, E, E1 <: E](effect: A < (Abort[E] & S)) extends AnyVal:
-
-    /** Handles the Abort[E] effect and returns its result as a `Choice`.
-      *
-      * @return
-      *   A computation that produces the result of this computation with Choice effect
-      */
-    def apply[ER]()(
-        using
-        ev: E => E1 | ER,
-        ct: SafeClassTag[E1],
-        tag: Tag[E1],
-        reduce: Reducible[Abort[ER]],
-        flat: Flat[A],
-        frame: Frame
-    ): A < (S & reduce.SReduced & Choice) =
-        Abort.run[E1](effect.asInstanceOf[A < (Abort[E1 | ER] & S)]).map(e => Choice.get(e.fold(_ => Nil)(List(_))))
-
-end SomeAbortToChoiceOps
-
-class SomeAbortToEmptyOps[A, S, E, E1 <: E](effect: A < (Abort[E] & S)) extends AnyVal:
-
-    /** Handles the Abort[E] effect translating it to an Abort[Absent] effect.
-      *
-      * @return
-      *   A computation that produces the result of this computation with Abort[Absent] effect
-      */
-    def apply[ER]()(
-        using
-        ev: E => E1 | ER,
-        ct: SafeClassTag[E1],
-        tag: Tag[E1],
-        reduce: Reducible[Abort[ER]],
-        flat: Flat[A],
-        frame: Frame
-    ): A < (S & reduce.SReduced & Abort[Absent]) =
-        Abort.run[E1](effect.asInstanceOf[A < (Abort[E1 | ER] & S)]).map {
-            case Result.Fail(_)        => Abort.get(Result.Fail(Absent))
-            case p @ Result.Panic(e)   => Abort.get(p.asInstanceOf[Result[Nothing, Nothing]])
-            case s @ Result.Success(a) => Abort.get(s.asInstanceOf[Result[Nothing, A]])
-        }
-end SomeAbortToEmptyOps
-
-class HandleSomeAbort[A, S, E, E1 <: E](effect: A < (Abort[E] & S)) extends AnyVal:
-
-    /** Handles the Abort[E] effect and returns its result as a `Result[E, A]`.
-      *
-      * @return
-      *   A computation that produces the a `Result[E, A]`
-      */
-    def apply[ER]()(
+class SomeAbortOps[A, S, E, E1 <: E](effect: A < (Abort[E] & S)) extends AnyVal:
+    def handled[ER](
```

Comment:
How about calling this `result`? Similar to ZIO's `exit`/`either`?

Reply:
I went through and updated a bunch of the name

handleAbort -> result
catchAbort -> catching
catchAbortPartial -> catchingPartial

someAbort[E].handled -> forAbort[E].result
someAbort[E].caught -> forAbort[E].catching
someAbort[E].caughtPartial -> forAbort[E].catchingPartial

handleEmptyAbort -> maybe